apps/explorer/lib/explorer/chain/gas.ex

```diff
@@ -6,5 +6,5 @@ defmodule Explorer.Chain.Gas do
   """
 
   @typedoc @moduledoc
-  @type t :: non_neg_integer()
+  @type t :: false | nil | %Decimal{:coef => non_neg_integer(), :exp => integer(), :sign => -1 | 1}
```

Comment:
why has this changed?

Reply:
on further investigation it seems this is the result of an upstream merge and probably incidentally included in whatever upstream patch added the revert reason functionality.